<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockIndexDetailsDialog</class>
 <widget class="QDialog" name="BlockIndexDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Block Index Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Chain Work</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelConf">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelVersionHex">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>0x00</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hash</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Bits</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidgetTransactions">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Version Hex</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="labelBits">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelMerkleRoot">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelVersion">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelHeight">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Next Block Hash</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="labelNonce">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Previous Block Hash</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Confirmations</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Merkle Root</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="labelPrevBlockHash">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="labelTime">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="labelMedianTime">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelChainWork">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="labelNextBlockHash">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonLoadTransactions">
     <property name="text">
      <string>Load Transactions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelHash">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="text">
      <string>00000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Median time</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Nonce</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
